### DISCUSSION TITLE
Adjusting upload media size

### DISCUSSION THREAD

**Discussion by nebulade** [pixelfed/pixelfed#5130] [Support]
Writing to seek some clarification on the configuration variables.
When attempting to change the media upload size, there are various variables in the play:
* env file with `MAX_PHOTO_SIZE` with a default of 15Mb
* in the UI at /i/admin/settings -> Media there is a `Max size` value
* database table `config_cache` which seems to reflect the settings UI value
* cache php file, which seems to be generated with values only from the `MAX_PHOTO_SIZE` env varible value

For a start, it seems this setting could be changed by env var as well as UI or are those two different settings?
If env var is not set at all, it always falls back to 15mb regardless what the db table hase.
If env var is set, updating the cache php file will have that value.

I guess my confusion comes from the fact, that if I disable the cache file via `ENABLE_CONFIG_CACHE=false` in the env file, the whole Settings UI is disabled with a hint to enable the config cache, but at least for this upload size value, those don't seem to be connected?

Any clarification on how to properly configure this (ideally not through env variable but through the Settings UI) would be highly appreciated.

**Comment by jdaviescoates** [comment]
Just to add, I've also found that regardless of what all the above settings are, until you also make correspondingly increased values inside ``php.ini`` changing those values doesn't actually work.

**Comment by vRobM** [comment]
Surely this is answerable.
